generators/generator-bot-template-generator/app/templates/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "<%= name %>",
+  "version": "0.0.1",
```

Comment:
I would just default to 1.0.0 for the file output by the generator.

Reply:
handled